src/main/java/com/fdmgroup/forex/models/User.java

```diff
@@ -0,0 +1,152 @@
+package com.fdmgroup.forex.models;
+
+import java.util.Date;
+import java.util.UUID;
+
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Temporal;
+import jakarta.persistence.TemporalType;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+
+/**
+ * Model a user account on the Forex platform
+ */
+@Entity
+public class User {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.UUID)
+	private UUID id;
+
+	@NotNull(message = "'username' must be present.")
+	@NotBlank(message = "'username' cannot be blank, i.e. is empty or contains only white spaces.")
+	@Column(unique = true, nullable = false)
+	private String username;
+
+	@NotNull(message = "'email' must be present.")
+	@Email(message = "'email' must be a valid Email address.")
+	@Column(unique = true, nullable = false)
+	private String email;
+
+	@NotNull(message = "'password' must be present.")
+	@NotBlank(message = "'password' cannot be blank, i.e. is empty or contains only white spaces.")
```

Comment:
Consider rewording to:
"'username' cannot be blank or contain only whitespace characters."

Reply:
Commit 5b60182 solved the issue